src/components/PicketSign/index.tsx

```diff
@@ -2,36 +2,25 @@ import { useEffect, useMemo, useState } from 'react';
 import { SignContent } from '../../types';
 import './PicketSign.css';
 
-export const PicketSign = ({ content }: { content: SignContent }) => {
-  const [flipped, setFlipped] = useState(true);
-  const [showContent, setShowContent] = useState(false);
```

Comment:
This exists to prevent the card contents from flashing when the page loads. However I think you're on to something. I'll make a suggestion below that can prevent the image flash without extra JS

Reply:
The suggestions feature in github won't let me suggest the entire code change, so I made #8  instead.